config/crds/apis.kcp.io_apiresourceschemas.yaml

```diff
@@ -48,6 +48,13 @@ spec:
                   Empty string means the core API group. \tThe resources are served
                   under `/apis/<group>/...` or `/api` for the core group."
                 type: string
+              nameValidation:
+                description: nameValidation can be used to configure name validation
+                  for bound crd. Allowed values are `DNS1123Subdomain` and `PathSegmentName`.
+                enum:
+                - DNS1123Subdomain
+                - PathSegmentName
+                type: string
```

Comment:
can we default this to the old behaviour?

Reply:
And add some docs what each means, maybe even put the regex there if it is not too long.

Reply:
Done.